---
layout: default
---
		<!-- Main -->
			<article id="main">
					
				<!-- One -->
					<section class="wrapper style4 container">
					
						<!-- Content -->
							<div class="content">
								<section>
										<header>
										<h3><strong>Miscellaneous projects
</strong></h3> </header>

<p>A place for all the one-off projects that don't merit their own page (yet). So far these projects have largely involved turning existing, but long forgotten, datasets into thoughtful and useful analyses. Currently these include (1) metagenomic data from hot springs in Kamchatka, Russia and (2) metagenomic data from aquariums. </p>


<div class="row">

	<div class="6u">
		<br>
		<br>
		<a class="image featured special"><img src="images/Ark_RussellNeches.jpg" alt="Photo of Arkashin Shurf, Kamchatka, Russia taken by Russell Neches; https://www.flickr.com/photos/rneches/" /></a>
	</div>

<div class="6u">

<header> <h4><strong>(1) Metagenomic data from two thermal pools in Kamchatka, Russia</strong></h4></header>
<p>This project was a collaboration with <a href="https://www.ecoevoint.ch/">Dr. Laetitia Wilkins</a>. Together we assembled draft metagenome-assembled genomes (MAGs)  from two hot springs located in the Kamchatka peninsula in Russia. We recovered 36 total MAGs including members that were taxonomically assigned to groups previously underrepresented in available genome data. This included several archaea (Korarchaeota, Bathyarchaeota and Aciduliprofundum) and one potentially new species within the bacterial genus Sulfurihydrogenibium. </p>


</div>

</div>


<header> <h4><strong>(2) Metagenomic data from aquariums</strong></h4></header>
<p>Previously, members of the Eisen lab sampled aquaria as part of an undergraduate research project that investigated microbial community assembly in the fall of 2012. As part of that work, metagenomes were generated from various aquaria. Later, using these metagenomes we were able to recover 11 MAGs reconstructed from freshwater and saltwater aquaria, including representatives of Polynucleobacter, Anaerolinea, Roseobacter, Flavobacteriia, Octadecabacter, Mycobacterium, and Candidate Phyla Radiation (CPR) members. </p>


<p>
<b>Related publications:</b>
<ul class="default">
<li><b>Ettinger CL</b>, Bryan J*, Tokajian S, Jospin G, Coil D & Eisen JA. 2021. Reconstruction of metagenome-assembled genomes from aquaria. Microbial Resource Announcements. DOI:<a href="https://journals.asm.org/doi/10.1128/MRA.00557-21"> 10.1128/MRA.00557-21</a> *undergraduate author </li>

<li><b>Ettinger CL*</b>, Wilkins LGE*, Jospin G & Eisen JA. 2019. Metagenome-assembled genomes provide new insight into the microbial diversity of two thermal pools in Kamchatka, Russia. Scientific Reports. DOI: <a href=https://www.nature.com/articles/s41598-019-39576-6>10.1038/s41598-019-39576-6</a> *co-first authors</li>		
</ul>	
</p>
		
												
									
								
							
								</section>
							</div>

					</section>
					
					
					
					
						
										
								
					
			</article>
