---
layout: default
---
		<!-- Main -->
			<article id="main">
					
				<!-- One -->
					<section class="wrapper style4 container">
					
						<!-- Content -->
							<div class="content">
								<section>
										<header>
										<h3><strong>Microbial communities of native wildflowers and invasive grasses
</strong></h3> </header>


<div class="row">

<div class="6u">
<a class="image featured special"><img src="images/GL_sampling_v2.jpg" alt="Native wildflowers; Photo credit: Andrew Latimer" /></a></div>

<div class="6u">
<p> While the rhizosphere is one of the most diverse terrestrial ecosystems and directly influences the cycling of nutrients within an ecosystem, relatively little is known about microbial community assembly and how plant species identify and interact with microbial species pools to determine the composition and abundance of the rhizosphere soil community.  </p>

</div>

</div>
		
		<p>I am collaborating with PhD candidate <a href="https://marinalaforgia.weebly.com/"> Marina LaForgia</a> in Susan Harrison and Andrew Latimer's labs at UC Davis, to study how below-ground plant-microbe feedback loops during competition between native wildflowers and invasive grass species in California grasslands across varying precipitation regimes. This study involves a field experiment which uses metagenomic sequencing to assess microbial community assembly and function during competition between communities of invasive grasses and native wildflowers grown in combination under varying precipitation regimes. It also involves a greenhouse experiment which uses high throughput sequencing of the 16S rRNA gene and internal transcribed spacer (ITS) regions to look at the rhizosphere communities of each grass / wildflower species when grown alone as well as in pair-wise combinations. 
						
</p>
		
												
									
								
							
								</section>
							</div>

					</section>
					
					
					
					
						
										
								
					
			</article>
