---
layout: post
comments: true

title: EDAMAME 2014's Greatest Hits
date: 2014-09-02 01:23:58.000000000 -07:00

---
This post was originally uploaded to the <a href="http://seagrassmicrobiome.org/"> seagrass microbiome website </a>.<br><br>

<p>I recently had the wonderful opportunity to attend the <strong><a href="http://edamame-course.org/">Explorations in Data Analyses for Metagenomic Advances in Microbial Ecology (EDAMAME) Workshop</a></strong>. The workshop was held at the Kellogg Biological Station in Michigan by Ashley Shade (@ashley17061), Tracy Teal (@<span class="u-linkComplex-target">tracykteal</span>) and Josh Herr (@number_three) - who are all AWESOME. I thought I'd go through and highlight what I found to be my favorite and/or most useful parts of the workshop.</p>
<p>Before I jump right in to the science, my favorite non-science related parts of EDAMAME included non-stop ice cream (lunch and dinner - with waffle cones!), meeting such a diverse crowd of microbe lovers, seeing <em>Guardians of the Galaxy</em> at the <strong><a  href="http://en.wikipedia.org/wiki/Alamo_Drafthouse_Cinema">Alamo Drafthouse Cinema</a></strong>, enjoying some local Michigan beer at Bell's Brewery and hearing Jack Gilbert (@gilbertjacka) play guitar and sing one night at the campfire (and yes, there were s'mores!).</p>
<p><img src="https://seagrassmicrobiome.files.wordpress.com/2014/08/img_4536.png" alt="IMG_4536" width="300" height="184" /></a></p>
<p>Anyway, pop this bad boy in and let's get started!</p>
<p><strong>Track 1: <a href="https://www.youtube.com/watch?v=yrotsEzgEpg">O-T-U Child </a></strong></p>
<p>For microbial ecology newbies, I think that EDAMAME's introduction to <strong><a href="https://edamame-course.github.io/docs/extra/Presentations/2014-08-14-AM_Ashley_Lecture2.pdf?raw=true">alpha diversity</a> </strong>and<strong> <a s href="https://edamame-course.github.io/docs/extra/Presentations/2014-08-14-PM_Ashley_Lecture3.pdf?raw=true">beta diversity</a></strong> lectures are a very good resource. I found the section on ordination plots especially helpful as even though I have taken introductory statistics classes such plots were never discussed! I would also point the microbial ecology newcomer to EDAMAME's <a  href="https://edamame-course.github.io/docs/the_shell.html"><strong>Introduction to Shell</strong></a> and Introduction to QIIME tutorials (<a  href="https://edamame-course.github.io/docs/intro_qiime_tutorial.html"><strong>Part 1</strong></a>, <strong><a  href="https://edamame-course.github.io/docs/qiime_2_tutorial.html">Part 2</a></strong> and <strong><a href="https://edamame-course.github.io/docs/qiime_3.html">Part 3</a></strong>) as both are very well documented and every step in the tutorials is well described.</p>
<p><em>Some things to think about:</em> <strong>Replication and Experimental Design. </strong>Something that was mentioned several times by different guest speakers (Pat Schloss (@PatSchloss) and Jim Cole, I think) was the idea of using a synthetic mock community each time you do a sequencing run to ascertain the error rate of that particular sequencing run. We also had several conversations about what replication means for microbial ecologists - if interested in what we discussed, go read: <strong><em><a  href="http://onlinelibrary.wiley.com/doi/10.1111/j.1462-2920.2010.02201.x/abstract;jsessionid=E21293EEE818ADF6773BFBB3CFC79348.f02t03?deniedAccessCustomisedMessage=&amp;userIsAuthenticated=false">Replicate or lie</a></em>.</strong></p>
<p><strong>Track 2: <a href="https://www.youtube.com/watch?v=wAiGd9nTGmM">Bioinformatics Killed My Computer</a></strong></p>
<p>If you have previously tried to pick OTU's or chimera check in QIIME or mothur (if confused, please review resources discussed in <strong>Track 1</strong>) on your personal laptop or even a lab computer, you are probably familiar with long wait times, black screens and spinning wheels of death.  Data sets are increasing in size at a much more rapid pace then the computational power included in standard use laptops. My Macbook Pro is five years old (which means its basically a dinosaur) and still kicking it with its 2.26 GHz Intel Core 2 Duo processor and its recently upgraded 8 GB of RAM. However, when it comes to some of my data analysis, its fan and slow response time make it sound like a small dying animal. So what is a researcher to do? Use a superior lab computer that has an i7 processor and 16 GB of RAM - YAY! But what happens when even that is not enough?</p>
<p><em>Solution Numero Uno: </em>Use a service like<strong> <a href="http://aws.amazon.com/">Amazon Web Services </a></strong>where you pay to use Amazon's computational power to run your analyses. Don't know how to set up an Amazon instance? The lovely EDAMAME instructors have got you covered with <strong><a href="https://edamame-course.github.io/docs/intro_to_ec2_instance.html">this tutorial</a> </strong>and also these follow-up tutorials on how to connect to your instance from a <strong><a  href="https://edamame-course.github.io/docs/mobaxterm.html">PC</a> </strong>or a <strong><a  href="https://edamame-course.github.io/docs/mac_os_linux_connect_to_ec2.html">Mac/Linux</a></strong> machine. One thing that makes using Amazon Web Services nice is that there are community API's you can use that have QIIME and/or mothur already installed.</p>
<p><em>Solution Numero Dos:</em> Use your university's computing cluster (if available). In my experience, most clusters are logged into the same way as you log into your Amazon instance - using SSH. For those unfamiliar with SSH, here is a <strong><a  href="https://www.digitalocean.com/community/tutorials/how-to-use-ssh-to-connect-to-a-remote-server-in-ubuntu">tutorial</a> </strong>I found using the magic of Google for Mac/Linux computers. One problem you might run into with using a cluster is that the software you want to use might not be installed - in which case, if you are lucky the computer gurus who manage your local cluster will install it and keep it up to date for you! However if you are unlucky... they might say that they won't install it at all.</p>
<p>If using either of these solutions, I highly suggest looking into using either <strong><a href="http://aperiodic.net/screen/quick_reference">Screen</a></strong> or <strong><a  href="https://gist.github.com/MohamedAlaa/2961058">Tmux</a> </strong>which were introduced to us at EDAMAME (I am currently using Tmux). Screen and Tmux allow you to open multiple bash windows on the machine you are SSH-ed into. This means if you run a command from inside a bash window in Screen/Tmux and then detach from Screen/Tmux and exit your SSH session, your command will continue to run on the server or cluster your were logged into. This means you no longer have to worry about the internet cutting out or leaving your personal computer on when running commands on remote server/clusters!</p>
<p><strong>Track 3: <a href="https://www.youtube.com/watch?v=eH3giaIzONA">I Wanna Download Some Genomes</a></strong></p>
<p>Have you ever wanted to download a genome from NCBI or MG-RAST but not wanted to hassle with the ever changing website interfaces? Have you ever wanted to download hundreds of genomes but the thought of all that clicking has you wanting to run to your safe place? Me too! I found guest lecturer, <a href="http://adina-howe.readthedocs.org/en/latest/"><strong>Adina Chuang Howe (@teeniedeenie)'s tutorial</strong> </a>on how to do these things from the shell to be one of the most useful parts of EDAMAME. The tutorial comes with the necessary scripts that you can re-purpose to download your favorite genomes - these could potentially save you hours and hours of valuable research time (and most of your sanity).</p>
<p><strong>Track 4: <a href="https://www.youtube.com/watch?v=Mrd14PxaUco">I Got A Database</a></strong></p>
<p>Throughout EDAMAME, we were introduced to several different marker gene databases (16S, ITS). The default database in QIIME is the <strong><a href="http://greengenes.lbl.gov/cgi-bin/nph-index.cgi">greengenes</a> </strong>database and until EDAMAME, I never realized just how many databases there were! I was especially surprised when in <strong><a  href="https://github.com/edamame-course/docs/blob/gh-pages/extra/Presentations/2014-08-19-Cole_EDAMAME_2014.pdf">Jim Cole's guest lecture</a></strong> he showed us a slide with a venn diagram comparing the sequences contained in the different fungal ITS databases and there was a definite visible discrepancy between what each database contained. I also found his slides comparing the taxonomic accuracy of the different fungal ITS databases as we have some seagrass fungal microbe data to analyze. I think the exposure to the different databases was helpful to my evolution as a microbial ecologist as it made me start thinking about which database I'm using for my analyses and what my biases these databases might introduce to my analysis. Some helpful questions to keep in mind when looking at different databases are laid out in this EDAMAME <strong><a  href="https://github.com/edamame-course/docs/blob/gh-pages/_posts/2014-08-19-database_questions.md">tutorial</a>.</strong></p>
<p><strong>Track 5: <a href="https://www.youtube.com/watch?v=CdqoNKCCt7A">Don't You (Forget To Visualize Me)</a></strong></p>
<p>Humans are very visually oriented creatures and having visually appealing, statistically accurate and reproducible graphs can really help a presentation succeed. A strong knowledge of R (which I someday hope to have) can help you achieve such graphs. We walked through several different <a href="https://github.com/edamame-course/docs/tree/gh-pages/extra/Jones_R_Tutorial_Files"><strong>R tutorials</strong> </a>relating to microbial ecology at EDAMAME which I thought were helpful when combined with EDAMAME's <strong><a href="https://edamame-course.github.io/docs/extra/Presentations/2014-08-14-PM_Ashley_Lecture3.pdf?raw=true">beta diversity</a> </strong>and <a  href="https://github.com/edamame-course/docs/blob/gh-pages/extra/Presentations/2014-08-18-AM_Ashley_Lecture4.pdf?raw=true"><strong>hypothesis testing</strong> </a>lectures. These R tutorials were also specifically aimed at utilizing OTU tables which helped me think about how I might apply what I was learning to my own data analysis. We were also introduced to a vast array of data visualization and exploration tools (<strong><a  href="https://github.com/edamame-course/docs/blob/gh-pages/extra/Presentations/2014-08-20-metagenomic_visualization.pdf?raw=true">see this lecture</a></strong>).</p>
<p><strong>Bonus Track: <a href="https://www.youtube.com/watch?v=PJQVlVHsFF8&amp;feature=kp">Hooked on Microbial Ecology</a> </strong>(because if I hadn't been hooked on microbial ecology before EDAMAME, I sure as would be now!)</p>
<p style="text-align:center;"> <img src="https://seagrassmicrobiome.files.wordpress.com/2014/09/bvioe_ecqai9ddj.jpg" width="400" height="264">
	<br>
	<em>EDAMAME Class of 2014 in our MoBio T-shirts!</em></p>
<p>All of the workshop materials for EDAMAME 2014 can be found on their <strong><a  href="edamame-course.org/">website</a></strong> and there is also a <strong><a  href="https://storify.com/ashley17061/edamame2014">storify</a></strong> of all the live tweeting that went on throughout the EDAMAME workshop using the hashtag #edamame2014. Also, look out for a future guest post on MoBio's blog, <strong><a href="http://www.mobio.com/blog/"><em>The Culture Dish</em></a></strong>, about seagrass microbes!</p>
