---
layout: post
comments: true

title: 'Thoughts on Fluidity and Inter-connectivity of Microbial Community Dynamics '
date: '2015-03-02T18:22:00.000-08:00'
tags:
- Microbial Ecology
- Advanced Soil Microbiology Class
- Microbiology
modified_time: '2015-03-02T18:22:00.823-08:00'
blogger_id: tag:blogger.com,1999:blog-955663505887648912.post-6610492995418440654
blogger_orig_url: http://everybodylovesscience.blogspot.com/2015/03/thoughts-on-fluidity-and-inter.html
---

This post is adapted from a blog post I wrote for my advanced soil microbiology graduate class.<br /><br />This post is in response to: <a href="http://journal.frontiersin.org/Journal/10.3389/fmicb.2012.00417/full">Shade,    Ashley, Hannes Peter, Steven D. Allison, Didier L. Baho, Mercè    Berga, Helmut Bürgmann, David H. Huber et al. "Fundamentals of    microbial community resistance and resilience."&nbsp;  Frontiers    in microbiology  &nbsp;3    (2012). </a></span></span> <br /><br />Three ideas stood out to me: that their can be multiple stable  states, the treatment of microbial communities as networks and the  description of communities as systems of genes and their functions. <br /><br /><b>Multiple Stable States&nbsp;</b><br />I  think that the idea that microbial communities might have multiple  stable states that they can shift between is really interesting. This  essentially lends weight to the notion that there is no one healthy  microbiome. Instead there are dozens upon dozens of microbial  communities that are "healthy". Also, microbial communities are composed  of lots of different microorganisms living in concert. If the cellist  suddenly dies (or leaves town), an orchestra will usually find another  cellist, have someone already in the orchestra stand in or shift the  music they play to not include cellos. The orchestra is no less stable,  just slightly different. <br /><br /><b>Microbial communities as networks</b><br />People  have been using network analysis to investigate complex traits and  molecular pathways for a while now. I think that the using microbial  co-occurrence to do network analysis on microbial communities is  brilliant. Network analysis could potentially allow you to find/see  functional groups of organisms (N fixers, etc) and might even allow you  to discover common microbial "teams" (different microbes often found  together). It'd be interesting to overlay functional information and  metadata into the network analysis and see what falls out.<br /><br /><b>Communities as Systems of Genes and their Functions</b><br />We've  talked a lot in this class about the dilemma of "who's there" vs. "what  are they doing". A lot of the reason that I think we are interested in  "who's there" is to try to either a) narrow in on the "who" to  investigate that "what" or b) to use the "who" to estimate the "what".  Genomics and high throughput sequencing techniques (amplicon gene  sequencing, metagenomics, metatranscriptomics, metametabolomics, etc)  allow us to more easily answer both the "who" and the "what". Although  such techniques are data heavy (and thus, could cause us to drown in  unanalyzed data), I think that these techniques will definitely be  influential if not transformative for the whole of microbial ecology.<br /><br /><br />