---
layout: post
comments: true

title: How Meta!
date: '2015-02-23T18:20:00.000-08:00'
tags:
- Advanced Soil Microbiology Class
- EDAMAME
- Genomics
- Metagenomics
- Data Analysis
modified_time: '2015-02-23T18:20:00.335-08:00'
blogger_id: tag:blogger.com,1999:blog-955663505887648912.post-8991644534345886579
blogger_orig_url: http://everybodylovesscience.blogspot.com/2015/02/how-meta.html
---

This post is adapted from a blog post I wrote for my advanced soil microbiology graduate class.<br /><br />In class we recently talked about metagenomics. I know that a lot of  the class missed out because of the soils conference, so I thought I'd  use this post to introduce a <a href="http://journal.frontiersin.org/Journal/10.3389/fpls.2014.00209/abstract">really good metagenomics review paper by Tom Sharpton from Oregon State.</a> It goes through the basic theory and approach to metagenomic analysis  and Table 1 is a really good resource for online metagenomic resources.<br /><br />I also suggest looking through <a href="https://edamame-course.github.io/docs/">these resources (or applying to go to this workshop).</a> I attended this past year and it was very informative for both amplicon  based sequencing (16S, ITS, etc) and metagenomics. All of the resources  from the workshop are online.<br /><br />Also the notes and slides from <a href="http://eukhits.wordpress.com/2014/08/10/esa-2014-workshop-materials-posted/">this ESA workshop </a>may prove useful to some.<br /><br />For those just getting into microbiology, our lab released a <a href="https://peerj.com/preprints/453/">"Swabs to Genomes" workflow </a>which  takes you from, well, swab to genome. The workflow includes culturing  microbes from the swab, isolating microbes, doing 16S Sanger sequencing  to determine microbial identity and then full genome sequencing and  assembly. Although this approach is not metagenomics, instead "simple"  genomics, it might still be helpful for those unfamiliar with assembly,  phylogenetics, etc.<br /><br /><br />