---
layout: post
comments: true

title: What is more important for classification - taxonomy or function?
date: '2015-02-16T18:18:00.000-08:00'
tags:
- Microbial Ecology
- Advanced Soil Microbiology Class
- Data Analysis
modified_time: '2015-02-16T18:18:00.249-08:00'
blogger_id: tag:blogger.com,1999:blog-955663505887648912.post-3189181716255779400
blogger_orig_url: http://everybodylovesscience.blogspot.com/2015/02/what-is-more-important-for.html
---

This post is adapted from a blog post I wrote for my advanced soil microbiology graduate class.<br /><br />Something I've been thinking a lot about recently is how we  characterize microbial communities. Now that sequencing is relatively  cheap, we can generate large amounts of data about the microbes of our  respective research projects. However, should we be performing amplicon  sequencing (16S, 18S, ITS) or metagenomics? <b>Should we be classifying communities by function or taxonomy?&nbsp;</b><br /><br />A recent short article by <b><a href="http://www.nature.com/ismej/journal/vaop/ncurrent/full/ismej2014157a.html">Xu et al</a></b>, discusses this very question. Xu et al used <b><a href="http://picrust.github.io/picrust/">PICRUST</a> </b>to  examine if either function or taxonomy was a better indicator of  microbial community composition. Using several publicly available  datasets, they compared 16S classification to functional classification  from PICRUST and in one instance to annotated metagenomic data. They  determined that adding functional information <b>DOES NOT</b> improve  classification accuracy of microbial communities. However, the authors  don't downplay the importance of functional information, emphasizing  that functional information may be very important depending on the  research question. The authors state that improving our ability to  classify samples into biologically relevant categories is not a reason  to obtain functional data over taxonomic data. So if the goal of your  research question is survey a microbial community, then metagenomics  might actually provide you with less useful data than plain old 16S rRNA gene sequencing for answering your question.&nbsp; <br /><a href="http://www.nature.com/ismej/journal/vaop/ncurrent/full/ismej2014157a.html"><br /></a><br /><div class="gs_citr" id="gs_cit0" tabindex="0"><b><a href="http://www.nature.com/ismej/journal/vaop/ncurrent/full/ismej2014157a.html">Xu,  Zhenjiang, et al. "Which is more important for classifying microbial  communities: whoâ€™s there or what they can do?" The ISME journal (2014).</a></b></div><br />